fda-model/paireddata/GraphicalUncertainPairedData.cs

```diff
@@ -151,7 +159,7 @@ public IPairedData SamplePairedData(double probability)
                 }
                 if (pairedData.RuleMap[nameof(pairedData.Xvals)].ErrorLevel > ErrorLevel.Unassigned)
                 {
-                    Array.Sort(pairedData.Xvals);//bad news.
+                    throw new Exception("");
```

Comment:
if you are going to do an exception we need to give it context - so add a message at least